.gitmodules

```diff
@@ -29,7 +29,7 @@
 	branch = mbedtls-2.16.8
 [submodule "libraries/abstractions/pkcs11/psa"]
 	path = libraries/abstractions/pkcs11/psa
-	url = https://github.com/Linaro/freertos-pkcs11-psa.git
+	url = https://github.com/lundinc2/freertos-pkcs11-psa.git
```

Comment:
Is this right? Do you want to change to your repo?

Reply:
No, this was a temporary change to test the regressions. I have a PR to upstream this, once that is merged, this is ready.

Reply:
Should this be in draft to keep CBMC from running?